spec/unit/resource/status_spec.rb

```diff
@@ -70,6 +70,15 @@
     expect(Puppet::Resource::Status.new(resource).resource).to eq("foo")
   end
 
+  it 'should set the provider_used correctly' do
+    expected_name = if Puppet.features.microsoft_windows?
```

Comment:
I believe that `Puppet::Util::Platform.windows?` is the way we're trying to do this check these days. @Iristyle has previously suggested that over the `Puppet.features.microsoft_windows?` way in tests.

Reply:
Updated.

@Iristyle is this true, should I open a ticket and convert 'Puppet.features.microsoft_windows?' to 'Puppet::Util::Platform.windows?' over in all of puppet?